en/exercises/network_mapping/considerations.md

```diff
@@ -0,0 +1,5 @@
+#### Considerations:
+
+### Shunning:
+	* If the target organization has IDS/IPS or Firewall in place, it is important to deal with it before conducting the mapping activity. In most cases, these activities are being blocked or either denied by these security technologies in placed. The reason behind why we should allow all traffic from our source machine is that to make sure that we are able to reach the target network to rule out the possibility of attackers being able to bypass these security technologies in parallel scans.
```

Comment:
It's important to explain where IDS/IPS are located, and where the auditor is located, outside or inside the network. I've seen few organizations using their IDS/IPS actively inside their LANs.